.github/workflows/knownprojects_test.yml

```diff
@@ -0,0 +1,12 @@
+name: KPDB - Test
+
+on:
+  workflow_call:
+
+jobs:
+  branch_test_build:
+    name: Build test run
+    uses: ./.github/workflows/knownprojects_build.yml
+    with:
+      run_export: true
```

Comment:
If we're exporting as part of a test, can we add a step to delete the outputs?

Reply:
would it be even better to split the export logic enough so that a test doesn't everything except upload?

I'd move everything that's currently before the upload into the "build" stage and set `run_export: false`

Reply:
the test now does everything except uploading (see [latest test run](https://github.com/NYCPlanning/data-engineering/actions/runs/6111936414/job/16588301689#step:7:243))